vendor/ocaml-blake3-mini/dune

```diff
@@ -4,8 +4,7 @@
  (foreign_archives blake3)
  (foreign_stubs
   (language c)
-  (names blake3_stubs)
-  (flags :standard -O3)))
```

Comment:
Is this relevant to this PR? If not, I suggest move it to a separate PR

Reply:
Yes, it is relevant. First we don't even set -O3 in the bootstrap so the development build is simply different. Secondly MSVC doesn't understand `-O3` and emits a warning when it encounters it. `-O2` as found in `:standard` however is fine.